types/react-map-gl/index.d.ts

```diff
@@ -186,6 +201,9 @@ export interface MarkerProps extends BaseControlProps {
     latitude: number;
     offsetLeft?: number;
     offsetTop?: number;
+    onDrag: (event: any) => void;
```

Comment:
Can we narrow down this type instead of any?

Reply:
Thank you for reviewing.
I think that the event arg is set [here](https://github.com/uber/react-map-gl/blob/4.0-release/src/utils/map-controller.js#L81).
And its type is defined as [MjolnirEvent](https://github.com/uber/react-map-gl/blob/4.0-release/flow-typed/npm/mjolnir.js_vx.x.x.js#L13) in the flow.
We can define the type to use MjolnirEvent as a reference as below.

``` TypeScript
interface MjolnirEvent {
    type: string,
    offsetCenter: {x: number, y: number},
    deltaX: number,
    deltaY: number,
    delta: number,
    scale: number,
    rotation: number,
    pointerType: string,
    metaKey: boolean,
    rightButton: boolean,
    stopPropagation: () => void,
    preventDefault: () => void,
    srcEvent: any
}
```

What do you think about this idea?

Reply:
Thank you for replying.
As the PR will be closed today, I added this just now.
I look forward to your PR.